src/components/ui/particles.tsx

```
@@ -0,0 +1,246 @@
+  alphaParticles?: boolean;
+  particleBaseSize?: number;
+  sizeRandomness?: number;
+  cameraDistance?: number;
+  disableRotation?: boolean;
+  className?: string;
+}
+
+const defaultColors: string[] = ["#ffffff", "#ffffff", "#ffffff"];
+
+const hexToRgb = (hex: string): [number, number, number] => {
+  hex = hex.replace(/^#/, "");
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)